frontend/src/components/elements/Metric/styled-components.ts

```diff
@@ -0,0 +1,59 @@
+/**
+ * @license
+ * Copyright 2018-2021 Streamlit Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import styled from "@emotion/styled"
+
+export const StyledText = styled.div(({ theme }) => ({
+  overflowWrap: "normal",
+  fontFamily: theme.genericFonts.bodyFont,
+  fontSize: theme.fontSizes.twoXS,
```

Comment:
![B9D4814D-1174-4D55-9204-86E638AE0819](https://user-images.githubusercontent.com/69432/126817066-2913e3dd-db1c-49e6-b3a7-24240fadb327.png)


Provided thoughts on the CSS. My only one check is that the label text at the top is INCREDIBLY small. Wondering why that might be the case. Wondering if this should be `smDefault` or something.

Reply:
Hmm I definitely had it smDefault and then Johannes said it was too big I think.